app/validators/uuid_validator.rb

```diff
@@ -0,0 +1,10 @@
+class UuidValidator < ActiveModel::EachValidator
+
+  UUID_REGEX = /\A[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\Z/i
```

Comment:
This is currently failing linting as it needs to be frozen:

```
app/validators/uuid_validator.rb:2:1: C: Layout/EmptyLinesAroundClassBody: Extra empty line detected at class body beginning. (https://rubystyle.guide#empty-lines-around-bodies)
app/validators/uuid_validator.rb:3:16: C: Style/MutableConstant: Freeze mutable objects assigned to constants.
  UUID_REGEX = /\A[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\Z/i
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

If you run `bundle exec rake lint` you can run the linting, you can do `bundle exec rake` to run both tests and linting sequentially.

Reply:
Took the linter's advice and applied /\A[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\Z/i.freeze.